src/Security/Authentication/Core/src/HandleRequestResult.cs

```diff
@@ -8,6 +8,10 @@ namespace Microsoft.AspNetCore.Authentication;
 /// </summary>
 public class HandleRequestResult : AuthenticateResult
 {
+    private static readonly HandleRequestResult _noResult = new() { None = true };
```

Comment:
Style: I think we use "NoResult" for class statics, and "_noResult" for class fields.